templates/_helpers.tpl

```diff
@@ -54,7 +64,9 @@ Create the name of the stackstorm-ha service account to use
 {{- end -}}
 {{- end -}}
 
-# Generate list of nodes for Redis with Sentinel connection string, based on number of replicas and service name
+{{/*
+Generate list of nodes for Redis with Sentinel connection string, based on number of replicas and service name
+*/}}
 {{- define "redis-nodes" -}}
```

Comment:
I didn't prefix this, because I'm not sure if we're overriding a subchart template helper.